vcl_templates/www.vcl.erb

```diff
@@ -220,7 +220,7 @@ sub vcl_recv {
     set req.http.GOVUK-ABTest-EducationNavigation = "A";
   } else if (req.url ~ "[\?\&]ABTest-EducationNavigation=B(&|$)") {
     set req.http.GOVUK-ABTest-EducationNavigation = "B";
-  } else if (req.url ~ "^/education(/|$)") {
+  } else if (req.url ~ "^/education(\/|\?|\$)") {
```

Comment:
I don't think `\$` should have the backslash - we're matching to the end of the string rather than a literal `$`.